types/types.go

```diff
@@ -552,9 +552,6 @@ func (s ServiceVolumeConfig) String() string {
 	if s.Volume != nil && s.Volume.NoCopy {
 		options = append(options, "nocopy")
 	}
-	if s.Volume != nil && s.Volume.Subpath != "" {
-		options = append(options, s.Volume.Subpath)
-	}
```

Comment:
Just to be sure, we have a warning in Compose to let the users know it won't be used?
What adding the warning here for all the other implementations?

Reply:
nope, Docker Compose **HAS** to use the mount API when `SubPath` or `CreateHostPath` is set